core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -405,6 +407,10 @@ struct SchedulerCountMetrics {
     num_dropped_on_age_and_status: usize,
     /// Number of transactions that were dropped due to exceeded capacity.
     num_dropped_on_capacity: usize,
+    /// Min prioritization fees in the transaction container
+    min_prioritization_fees: u64,
+    // Max prioritization fees in the transaction container
```

Comment:
Other fields are all doc-comments, let's be consistent:
```suggestion
    /// Max prioritization fees in the transaction container
```